packages/grafana-ui/rollup.config.ts

```diff
@@ -1,6 +1,6 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
-import image from '@rollup/plugin-image';
```

Comment:
Please could you remove `@rollup/plugin-image` from the `grafana-ui/package.json` if it is now unused?

Reply:
oh yes, definitely!